app/assets/javascripts/mixins/ScrollToOnMountMixin.coffee

```diff
@@ -0,0 +1,12 @@
+@ScrollToOnMountMixin =
+  componentDidMount: ->
+    $('body').animate scrollTop: @positionInPage() - @marginTop(), 700
```

Comment:
Je trekt strings van elkaar af. Dat kan in javascript maar ik vind het heel vies, aangezien een + juist weer string concatenatie is en het heel verleidelijk is achter deze berekening een + te zetten voor een offset oid.